fhir-persistence/src/test/java/com/ibm/fhir/persistence/test/common/AbstractWholeSystemSearchTest.java

```diff
@@ -15,23 +15,25 @@
 import java.util.List;
 import java.util.Map;
 
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
-import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.resource.Basic;
 import com.ibm.fhir.model.resource.Resource;
 import com.ibm.fhir.model.type.Canonical;
 import com.ibm.fhir.model.type.Code;
 import com.ibm.fhir.model.type.Coding;
 import com.ibm.fhir.model.type.Meta;
 import com.ibm.fhir.model.type.Uri;
 
-public abstract class AbstractSearchAllTest extends AbstractPersistenceTest {
-    protected String patientId;
-    protected String lastUpdated;
+public abstract class AbstractWholeSystemSearchTest extends AbstractPersistenceTest {
```

Comment:
Even if we are not using it, I think there is a benefit from minimally assigning a group to the test.  

Not something that must change... just to think about.